ml-operator/values.yaml

```diff
@@ -0,0 +1,109 @@
+
+nodeSelector: {}
+affinity: {}
+tolerations: {}
+
+ml-operator:
+  enabled: true
+  image: mojaloop/ml-operator:v0.1.6
+  env:
+    NODE_ENV:
+      name: NODE_ENV
+      value: production
+    LOG_LEVEL:
+      name: LOG_LEVEL
+      value: debug
```

Comment:
should this also not be "info" by default?

Reply:
Yep